osutils.pm

```diff
@@ -74,4 +77,20 @@ sub qv($) {
 }
 ## use critic
 
+# runcmd use SIGCHLD handler to wait all child processes at exit and reap zombies
+sub runcmd {
+    diag "running " . join(' ', @_);
+    my %children;
+    local $SIG{CHLD} = sub {
+        local ($!, $?);
+        while ((my $pid = waitpid(-1, WNOHANG)) > 0) {
+            delete $children{$pid};
+            cleanup_child($pid, $?);
```

Comment:
I can't find ```cleanup_child()``` in the [codebase,](https://github.com/os-autoinst/os-autoinst/search?utf8=%E2%9C%93&q=cleanup_child&type=) am i missing something?

Reply:
seems I can't get pid of `system` with `waitpid`, is it possible that Perl reap system subprocesses itself?
if so to use `local $SIG{CHLD} = 'DEFAULT'` should be ok here?